auth-web/src/components/auth/account-deactivate/DeactivateCard.vue

```diff
@@ -21,35 +21,34 @@
 </template>
 
 <script lang="ts">
-import { Component, Prop } from 'vue-property-decorator'
+import { computed, defineComponent, ref } from '@vue/composition-api'
 import Vue from 'vue'
+import { useI18n } from 'vue-i18n-bridge'
 
-@Component({
-})
-export default class DeactivateCard extends Vue {
-  @Prop() private type: string
-
-  // no type matches means show it for all types
-  private infoArray: { title: string, description?: string, type?: string }[] = [
-    {
-      title: 'deactivateMemberRemovalTitle',
-      description: 'deactivateMemberRemovalDesc'
-    },
-    {
-      title: 'businessRemovalTitle',
-      description: 'businessRemovalDesc'
-    },
-    {
-      title: 'padRemovalTitle',
-      type: 'PREMIUM'
-    }
+export default defineComponent({
+  name: 'DeactivateCard',
+  props: {
+    type: String
+  },
+  setup (props) {
+    const { t } = useI18n()
+    const infoArray = ref([
```

Comment:
I don't think you need a ref for this since this array is static right?

Reply:
made the change